curriculum/challenges/english/10-coding-interview-prep/rosetta-code/execute-a-markov-algorithm.md

```diff
@@ -85,55 +85,53 @@ assert(typeof markov === 'function');
 
 ```js
-assert.deepEqual(markov(rules[1], datas[1]), outputs[1]);
+assert.deepEqual(markov(rules[1], data[1]), outputs[1]);
 ```
 
 `markov(["A -> apple","WWWW -> with","Bgage -> ->.*","B -> bag","->.* -> money","W -> WW","S -> .shop","T -> the","the shop -> my brother","a never used -> .terminating rule"],"I bought a B of As W my Bgage from T S.")` should return the string `I bought a bag of apples with my money from T shop.`.
 
 ```js
-assert.deepEqual(markov(rules[2], datas[2]), outputs[2]);
+assert.deepEqual(markov(rules[2], data[2]), outputs[2]);
 ```
 
 `markov(["_+1 -> _1+","1+1 -> 11+","1! -> !1",",! -> !+","_! -> _","1*1 -> x,@y","1x -> xX","X, -> 1,1","X1 -> 1X","_x -> _X",",x -> ,X","y1 -> 1y","y_ -> _","1@1 -> x,@y","1@_ -> @_",",@_ -> !_","++ -> +","_1 -> 1","1+_ -> 1","_+_ -> "],"_1111*11111_")` should return the string `11111111111111111111`.
 
 ```js
-assert.deepEqual(markov(rules[3], datas[3]), outputs[3]);
+assert.deepEqual(markov(rules[3], data[3]), outputs[3]);
 ```
 
 `markov(["A0 -> 1B","0A1 -> C01","1A1 -> C11","0B0 -> A01","1B0 -> A11","B1 -> 1B","0C0 -> B01","1C0 -> B11","0C1 -> H01","1C1 -> H11"],"")` should return the string `00011H1111000`.
 
 ```js
-assert.deepEqual(markov(rules[4], datas[4]), outputs[4]);
+assert.deepEqual(markov(rules[4], data[4]), outputs[4]);
 ```
 
 # --seed--
 
 ## --after-user-code--
 
 ```js
-
 let rules=[["A -> apple","B -> bag","S -> shop","T -> the","the shop -> my brother","a never used -> .terminating rule"],
             ["A -> apple","B -> bag","S -> .shop","T -> the","the shop -> my brother","a never used -> .terminating rule"],
             ["A -> apple","WWWW -> with","Bgage -> ->.*","B -> bag","->.* -> money","W -> WW","S -> .shop","T -> the","the shop -> my brother","a never used -> .terminating rule"],
             ["_+1 -> _1+","1+1 -> 11+","1! -> !1",",! -> !+","_! -> _","1*1 -> x,@y","1x -> xX","X, -> 1,1","X1 -> 1X","_x -> _X",",x -> ,X","y1 -> 1y","y_ -> _","1@1 -> x,@y","1@_ -> @_",",@_ -> !_","++ -> +","_1 -> 1","1+_ -> 1","_+_ -> "],
             ["A0 -> 1B","0A1 -> C01","1A1 -> C11","0B0 -> A01","1B0 -> A11","B1 -> 1B","0C0 -> B01","1C0 -> B11","0C1 -> H01","1C1 -> H11"]];
-let datas=["I bought a B of As from T S.",
-            "I bought a B of As from T S.",
-            "I bought a B of As W my Bgage from T S.",
-            "_1111*11111_",
-            "000000A000000"];
+let data=["I bought a B of As from T S.",
+           "I bought a B of As from T S.",
+           "I bought a B of As W my Bgage from T S.",
+           "_1111*11111_",
+           "000000A000000"];
 let outputs=["I bought a bag of apples from my brother.",
-            "I bought a bag of apples from T shop.",
-            "I bought a bag of apples with my money from T shop.",
-            "11111111111111111111",
-            "00011H1111000"]
-
```

Comment:
why remove this newline?

Reply:
All other code block above and below have no empty first/last lines.
Should I readd it?

Reply:
I don't think removing this line fixes any misspelled words

Reply:
Reverted.